ptf_tests/common/lib/__init__.py

```diff
@@ -1,3 +1,20 @@
+# Copyright (c) 2022 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#

```

Comment:
Lots of empty lines in all the files in this commit. This can be cleaned up to reduce the license block